test/Governance.test.ts

```diff
@@ -169,30 +174,129 @@ describe('Governance', function () {
         proposalCalldatas,
         descriptionHash
       )
-      await this.governance.connect(this.voter).castVote(proposalId, 1)
-      await this.governance.connect(this.delegatee1).castVote(proposalId, 1)
-      await this.governance.connect(this.delegatee2).castVote(proposalId, 1)
-      await this.governance.connect(this.delegatee3).castVote(proposalId, 1)
-      await this.governance.connect(this.newDelegatee).castVote(proposalId, 1)
+      expect(await this.governance.state(proposalId)).to.equal(0)
```

Comment:
question: is `proposalId` always going to be `0`?

Is this expect passing as the helper method is only ever called once per a `it`?

Reply:
Its not proposalId, proposalId will be some kind of random hash from proposal data. Its state initially though will always be 0 after proposal creation.

Reply:
Added clarification in code.